GitUpKit/Extensions/NSView+Embedding.h

```diff
@@ -0,0 +1,17 @@
+//
+//  NSView+Embedding.h
+//  GitUpKit (OSX)
+//
+//  Created by Dmitry Lobanov on 06.04.2020.
+//
+
+#import <AppKit/AppKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSView (Embedding)
+- (void)embedView:(NSView *)view;
++ (void)embedView:(NSView *)view inView:(NSView *)superview;
```

Comment:
Just keep `-embedView:`, this second one isn't really used.

Reply:
Should I remove both declaration and implementation of class method?

Reply:
No, I'm just saying only the first one is being used externally. Don't expose API that nobody is using is a generally good rule of thumb.